pdfminer/pdfinterp.py

```diff
@@ -146,13 +149,14 @@ def copy(self) -> "PDFGraphicState":
         obj.flatness = self.flatness
         obj.scolor = self.scolor
         obj.ncolor = self.ncolor
+        obj.ocg = self.ocg
```

Comment:
Storing it here in the `PDFGraphicState` is ok, because this is pdfminer.six' internal representation of the PDF. 

Would only changing pdfinterp.py suit your usecase?